src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,199 @@
+
+            // Initialize the app
+            Initialize();
+
+            // Create the UI
+            CreateLayout();
+        }
+
+        private void CreateLayout()
+        {
+            // Create a new layout for the entire page
+            var layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Add the map view to the layout
+            layout.AddView(_myMapView);
+
+            // Show the layout in the app
+            SetContentView(layout);
+        }
+
+        private void Initialize()
+        {
+            // Center the map on San Diego.
+            Map map = new Map(Basemap.CreateStreets());
+            _myMapView.Map = map;
+            _myMapView.SetViewpointCenterAsync(32.727, -117.1750, 40000);
+
+            // Create a ClosestFacilityTask using the San Diego Uri.
+            _task = ClosestFacilityTask.CreateAsync(_sanDiegoServiceAreaUri).Result;
+
+            // List of facilities to be placed around San Diego area.
+            _facilities = new List<Facility> {
+                new Facility(new MapPoint(-1.3042129900625112E7, 3860127.9479775648, _spatialReference)),
+                new Facility(new MapPoint(-1.3042193400557665E7, 3862448.873041752, _spatialReference)),
+                new Facility(new MapPoint(-1.3046882875518233E7, 3862704.9896770366, _spatialReference)),
+                new Facility(new MapPoint(-1.3040539754780494E7, 3862924.5938606677, _spatialReference)),
+                new Facility(new MapPoint(-1.3042571225655518E7, 3858981.773018156, _spatialReference)),
+                new Facility(new MapPoint(-1.3039784633928463E7, 3856692.5980474586, _spatialReference)),
+                new Facility(new MapPoint(-1.3049023883956768E7, 3861993.789732541, _spatialReference))
+                };
+
+            // Create a symbol for displaying facilities.
+            _facilitySymbol = new PictureMarkerSymbol(_pictureMarkerUri);
+            _facilitySymbol.Height = 30;
+            _facilitySymbol.Width = 30;
+
+            // Incident symbol
+            _incidentSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Cross, System.Drawing.Color.FromArgb(255, 0, 0, 0), 20);
+
+            // Route to hospital symbol
+            _routeSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.FromArgb(255, 0, 0, 255), 2.5f);
+
+            // Create Graphics Overlays for incidents and facilities.
+            _incidentGraphicsOverlay = new GraphicsOverlay();
+            _facilityGraphicsOverlay = new GraphicsOverlay();
+
+            // Create a graphic and add to graphics overlay for each facility.
+            foreach (Facility facility in _facilities)
+            {
+                _facilityGraphicsOverlay.Graphics.Add(new Graphic(facility.Geometry, _facilitySymbol));
+            }
+
+            // Add each graphics overlay to MyMapView.
+            _myMapView.GraphicsOverlays.Add(_facilityGraphicsOverlay);
```

Comment:
I recommend swapping these two lines and adding the facility overlay last. As is, the route line is rendered on top of the hospital icon.